pkg/ocm/ocm.go

```diff
@@ -40,6 +40,21 @@ var ccamServiceLog = slTemplate{
 	InternalOnly: false,
 }
 
+type limitedSupportReasonTemplate struct {
+	Details string
+	Summary string
+}
+
+var chgmLimitedSupport = limitedSupportReasonTemplate{
+	Details: "Action required: cluster not checking in",
+	Summary: "Your cluster is no longer checking in with Red Hat OpenShift Cluster Manager. Possible causes include stopping instances or a networking misconfiguration. If you have stopped the cluster instances, please start them again - stopping instances is not supported. If you intended to terminate this cluster then please delete the cluster in the Red Hat console",
+}
+
+var ccamLimitedSupport = limitedSupportReasonTemplate{
+	Details: "Action required: Restore missing cloud credentials",
+	Summary: "Your cluster requires you to take action because Red Hat is not able to access the infrastructure with the provided credentials. Please restore the credentials and permissions provided during install",
+}
```

Comment:
Should the above `slTemplates` be removed since we're now placing clusters into limited support instead?